### ISSUE TITLE
AustLII translator modifications needed

### ISSUE THREAD

**Issue by LawData-user** [zotero/translators#3389]
The translator does not change the various version of the Federal circuit and Family Court name, which I suspect needs to be added after line 114 in the code as written, eg

"Federal Circuit and Family Court of Australia - Division 1 Appellate Jurisdiction" : "FedCFamC1A"
"Federal Circuit and Family Court of Australia - Division 1 First Instance" : "FedCFamC1F"
"Federal Circuit and Family Court of Australia - Division 2 First Instance" : "FedCFamC2F"

However, the translator is broken across many courts as it does not capture the abbreviation rather than the court name because it is looking in the wrong part of the page.

It would also be great if it would recognise the web address that begins with "classic."  and not just the addresses that start with "www8."  Aside from the starting word the urls are the same.  BUT if you only have time to fix one thing please make it the media neutral citation abbrieiation.

I would be really helpful if this could be fixed as it is very tedious always having to manually alter the capture data.

thanks.  G

**Comment by LawData-user** [comment]
Thank you for such a quick response. Sorry I didn’t post forum first - had seen old posts and thought GitHub was where it needed raising - will post on forum first in future. Will test out the changes this evening (am in Aust so at work now) and let you know if I find any difficulties. I really appreciate your work on these things. RegardsGrantOn Nov 21, 2024, at 3:05 AM, Abe Jellinek ***@***.***> wrote:﻿
Thanks! Fixed. In the future, example URLs are helpful, and please make a forum post first - a Zotero developer can figure out what's going on and create GitHub issues.

**Comment by LawData-user** [comment]
Hi Abe,

Just tried AGLC4 to insert a case citation where the record has both the
medium neutral scraped form AustLII and the reported citation that I added
later.  The inserted citation came up as

*Stanford v Stanford* (2012) 247 CLR HCA 52, 108.

The Correct citation should have been *Stanford v Stanford* (2012) HCA 52;
(2012) 247 CLR 108.

Not sure if this is hard to fix?

thanks again

Grant

**Comment by LawData-user** [comment]
Hi AbeNo problems. I really appreciate the work you are doing and understand there needs to be some structure to the processes. The AUSTLII translator and AGLC curator have come along remarkably over the last 10 years. My regret is that I haven’t learned to assist with coding so I feel a little embarrassed I am not contributing. I am an expert in law so feel free to contact me if you have queries about how citations should be composed - it’s the least I can do. Thanks again for your work and time. RegardsGrantOn Nov 22, 2024, at 5:59 AM, Abe Jellinek ***@***.***> wrote:﻿
Sorry, that may have come off as a bit harsh. I don't mean to be rude, it's just difficult to keep track of translator support requests when they're coming in from new GitHub issues, from comments on old issues and PRs, and so on, instead of following our standard reporting procedure, which starts on the forums. I'm happy to help with issues like the one you reported here, but zotero/translators GitHub issues aren't the right place to ask for support with anything that goes beyond the item data produced by the translator. Nobody who works on this repo is a legal citations expert (that I know of); some people on the forums are.

**Comment by LawData-user** [comment]
Hi Abe,

The translator is working well for me on AustLII.

I got frustrated with the AGLC4 .csl so took to it with Claude.ai to make
it work better.  Have attached the revised version of it.

I have posted on the forum (AGLC 4 and dual case cites - Zotero Forums
<https://forums.zotero.org/discussion/comment/479801#Comment_479801>) but
no developer responses yet.

Where should I post this, assuming it is helpfull?

Thanks again.

Grant
